cmd/workflow-trigger/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/codingpop/workflow-trigger"
+)
+
+var (
+	owner       string
+	repo        string
+	accessToken string
+	eventType   string
+)
+
+func init() {
+	flag.StringVar(&owner, "owner", "", "Repository owner")
+	flag.StringVar(&repo, "repo", "", "GitHub repository name")
+	flag.StringVar(&accessToken, "access-token", "", "Personal access token")
+	flag.StringVar(&eventType, "event-type", "", "A custom webhook event name")
+}
+
+func main() {
+	flag.Parse()
+
+	if owner == "" {
+		log.Fatalln("must provide repository owner")
+	}
+	if repo == "" {
+		log.Fatalln("must provide repository name")
+	}
+	if accessToken == "" {
+		log.Fatalln("must provide personal access token")
+	}
+	if eventType == "" {
+		log.Fatalln("must provide event type")
+	}
```

Comment:
If you change the variables as suggested above, they will be pointers, but they're never nil, so you can deference them here.
```suggestion
	if *owner == "" {
		log.Fatalln("must provide repository owner")
	}
	if *repo == "" {
		log.Fatalln("must provide repository name")
	}
	if *accessToken == "" {
		log.Fatalln("must provide personal access token")
	}
	if *eventType == "" {
		log.Fatalln("must provide event type")
	}
```